log/sequencer.go

```diff
@@ -22,6 +23,11 @@ type Sequencer struct {
 	timeSource util.TimeSource
 	logStorage storage.LogStorage
 	keyManager crypto.KeyManager
+
+	// These parameters can be safely adjusted during operation
```

Comment:
I'm deferring that question until we define a proper admin API. At the moment they won't be changed while the log is running. I want to get the related storage API changes in while we only have one implementation.